pkg/symbolizer/symtab_linux.go

```diff
@@ -251,7 +277,13 @@ func (e *symbolTable) lookupByAddr(address uint64) string {
 }
 
 // getRuntimeBaseAddr gets the runtime base address of the main executable from /proc/pid/maps
-func getRuntimeBaseAddr(pid uint32) (uint64, error) {
+func getRuntimeBaseAddr(task Task, table *symbolTable, pid uint32) (uint64, error) {
+	if runtimeBaseAddr := table.runtimeBaseAddrCache[task.BaseAddrHash]; runtimeBaseAddr != 0 {
```

Comment:
I suppose runtimeBaseAddr can never be zero so we don't need to distinguish between not found and 0, right?